### PR TITLE
update audit

### PR DESCRIPTION
**Pull request by MathisGD**
- our answer for the high vulnerability has been added (the previous report doesn't contain any of the answers / fixes)

### CODE REVIEW

File: audits/2024-01-05-morpho-blue-cantina-competition.pdf
Comment (QGarchery) [inline audits/2024-01-05-morpho-blue-cantina-competition.pdf:1]: > our answer for the high vulnerability has been added (the previous report doesn't contain any of the answers / fixes)

Do you know if they plan to include our comments for the medium risk issues ?

The 2 other minor changes are:
- changed title from "Morpho Blue Competition" to "Morpho Blue Security Review". Should we change the title of that file then ?
- removed the line "The present report only outlines the critical, high and medium risk issues." in section 2 (but it still applies)

**Review by QGarchery — DISMISSED** [review]

File: audits/2024-01-05-morpho-blue-cantina-competition.pdf
Comment (MathisGD) [reply audits/2024-01-05-morpho-blue-cantina-competition.pdf:1]: >Do you know if they plan to include our comments for the medium risk issues ?

yes, they just sent it to me and I pushed it.

Comment (MathisGD) [reply audits/2024-01-05-morpho-blue-cantina-competition.pdf:1]: >The 2 other minor changes are:

Thanks for highlighting, and i think that it's fine to keep it like that.

**Review by QGarchery — APPROVED** [review]
Every medium has Morpho Labs response, except:
- 3.2.5 & 3.2.6, both fixed with a new computation of repaidShares in liquidate
- 3.2.8, where I think that it had one fix with setting a smaller max rate (also monitoring markets at creation)

**Pull request merged**